### ISSUE TITLE
Allow queueing prompts

### ISSUE THREAD

**Issue by dwt** [divamgupta/diffusionbee-stable-diffusion-ui#25]
This would likely need something like #24 first, or it would lead to a lot of lost images. But right now the wait time during image generation is nicely used to queue the next image for generation.

**Comment by dongphuchaitrieu** [comment]
I would love to try this feature!

**Comment by elbowdonkey** [comment]
+1 from me. In fact, I don't know of a single "all in one" Stable Diffusion project that does this, and it seems trivial to do.

**Comment by matthewmayer** [comment]
Agree this would be great, as you could queue up 10-20 prompts to run overnight rather than sit there waiting for each one to finish.

**Comment by TheCatFace** [comment]
+1. I was just coming here to suggest this.

**Comment by nemeth-it** [comment]
+1

That was the first thing I was missing. It would especially help to generate your prompt against multiple Styles. A queue feature would be awesome!

**Comment by joshdance** [comment]
I asked for this on Twitter, came here to ask for it again. Would greatly help my workflow and experimentation.

**Comment by joserodolfofreitas** [comment]
+1. It would be great to queue some prompts and let the app run during the night.

**Comment by barryanders** [comment]
> +1. Ideally, users can queue each prompt with specific options, but just queueing the prompt itself would be great already. Another possible way to implement this without too many UI implications would be to allow users to import a file containing the multiple prompts.

Agreed that users should be able to use a text file to queue prompts with custom settings per prompt as desired. The syntax doesn't really matter, but maybe something like:

```
prompt|num images|image height|image width|steps|batch size|guidance scale|seed
```

```
"hyperrealistic obese luke skywalker"|100|512|512|25|1|7.5|-1
"hyperrealistic obese leia organa"|100|512|512|25|1|7.5|-1
```

**Comment by Polyculturerawks** [comment]
Second this. I have seen this requested multiple times on the Discord server as well :)
